code/style.css

```diff
@@ -5,3 +5,432 @@ this can be used for the FAQ-accordion. read more here: https://www.w3schools.co
+  border-radius: 0.5em 0.5em 0 0;
+  
+}
+
+.sign-up {
+  font-family: 'Monofett', cursive;
+  position: sticky;
+  font-size: 40px;
+  margin: 0 0 20px 0;
+  text-align: center;
+  color: #8413a0;
+}
+
+.say-what {
+  margin-top: 10px;
+  color: #489268;
+}
+
+button {
+  width: 150px;
+  height: 50px;
+  background-color: #8413a0;
+  align-self: center;
+  border: 2px solid black;
+  border-radius: 1em;
+  filter: drop-shadow(5px 5px rgb(29, 106, 150));
+  
+}
+
+button:hover {
+  box-shadow: 5px 5px  #F5DC5A,
+  -0px -0px  #ffffff;
+  filter: none;
+  
+}
+
+input { 
+  border-radius: 0.5em;
+}
+
+select {
+  border-radius: 0.5em;
+}
+
+.footer-list {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+  color: brown;
+  margin: 20px 0 50px 0;
+  padding-top: 20px;
+  list-style-type: none;
+}
+
+.footer-list li a {
+  color: #8413a0c0;
+}
+
+
+
+
+/* -------tablet sizing --------- */
+
+
+@media (min-width: 769px) and (max-width: 1097px) {
+
+
+.hero-text {
+  font-size: 8em;
+}
+
+nav {
+  position: absolute;
+  display: flex;
+  background-color: transparent;
+  align-items: center;
+  top: 82%;
+  box-shadow: none;
+}
+
+.navbar-links {
+  display: flex;
+}
+
+.navbar-links ul {
+  flex-direction: row;
+  text-decoration: none;
+}
+
+.navbar-links li {
+  /* border: 2px solid red; */
+  list-style-type: none;
+  text-decoration: none;
+  font-size: 1.5em;
+  padding: 0 40px 0 40px;
+}
+
+.navbar-links li a {
+  color: #8413a0c0;
+}
+
+.toggle-btn {
+  display: none;
+}
+
+.site-info {
+  font-size: 150%;
+}
+
+.forms-container {
+  flex-direction: row;
+  justify-content: space-between;
+  margin: 20px 20px 70px 20px;
+}
+
+form {
+  border-radius: 15%;
+  margin: 10px;
+}
+
+.accordion {
+  border-radius: 15%;
+  margin: 10px;
+}
+
+.footer-list {
+  font-size: 22px;
+  padding-top: 20px;
+}
+
+}
+
+/* -----desktop sizing------ */
+
+
+@media (min-width: 1098px) {
+
+  header {
+    color: #F5DC5A;
+  }
+
+  nav {
+    background-color: transparent;
+    position: absolute;
+    display: flex;
+    padding: 20px;
+    top: 82%;
+    box-shadow: none;
+  }
+
+  .navbar-links {
+    display: flex;    
+}
+  
+  .navbar-links ul {
+    text-decoration: none;
+    flex-direction: row;
+    justify-content: center;
+  }
+  
+  .navbar-links li {
+    list-style-type: none;
+    text-decoration: none;
+    font-size: 2em;
+    padding: 0 80px 0 80px;
+    }
+
+  .navbar-links li a {
+    color: #8413a0c0;
+    }
+
+  .toggle-btn {
+    display: none;
+    }
+
+  .hero-text {
+    font-size: 1000%;
+    top:10%;
+  }
+
+  .site-info {
+    font-size: 40px;
+  }
+
+
+  .forms-container {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    margin: 100px 0 20px 0;
+  }
+
+  form {
+    width: 500px;
+    height: 700px;
+  }
+
+  .accordion {
+    width: 500px;
+    height: 700px;
+  }
+
+  h1 {
+    font-size: 40px;
+  }
+
+  h3 {
+    font-size: 25px;
+  }
+
+}
```

Comment:
Good approach using the mobile first approach and then using media queries to style for tablet and desktop view!